react-components/src/apps/messages.js

```diff
@@ -86,6 +86,7 @@ const intlData = {
             "the wiki URL field once you fill this field. Replace everything inside \\{\\{\\}\\}.",
         appDocumentation: "DE App Documentation",
         documentationTemplate: "View Documentation Template",
+        htcondorExtraRequirements: "HT Condor Extra Requirements",
```

Comment:
Extremely tiny but should probably be "HTCondor" (without the space), that's what their stuff uses.